### PR TITLE
Adding address to national registry

### PR DESCRIPTION
**Pull request by dabbeg**
## What

Making address info available in separable form

## Why

To prefill a application form with each of the attributes I added

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Comment by obmagnusson** [comment]
This will likely break how the Service portal is displaying this information. Would think the PR runner should catch that.

File: libs/api/domains/national-registry/src/lib/nationalRegistry.service.ts [hunk 1]
```diff
@@ -26,7 +26,15 @@ export class NationalRegistryService {
...
+  async getLegalResidence(houseCode: User['houseCode']): Promise<string> {
+    const address = await this.nationalRegistryApi.getAddress(houseCode)
+    if (!address) {
+      return ''
+    }
+    return `${address.streetAddress}, ${address.postalCode} ${address.city}`
+  }
+
+  getAddress(houseCode: User['houseCode']) {
+    return this.nationalRegistryApi.getAddress(houseCode)
```
Comment (darrikonn) [inline libs/api/domains/national-registry/src/lib/nationalRegistry.service.ts:38]: Did you have a look at if the soap client is caching anything (wondering about two requests made)

**Review by darrikonn — APPROVED** [review]

**Comment by darrikonn** [comment]
> This will likely break how the Service portal is displaying this information . Would think the PR runner should catch that.

No he keeps the `legalResidence` field which is used by the service portal. He's just adding an additional object field (`address`)

**Comment by darrikonn** [comment]
Closing due to outdated api

**Pull request closed without merging**